### PR TITLE
Editor: Signal Connection Dock improvements

### PR DESCRIPTION
**Pull request by dalexeev**
* Add icons.
* Add "Open Documentation" context menu item.

### CODE REVIEW

File: editor/connections_dialog.cpp [hunk 1]
```diff
@@ -1130,7 +1138,7 @@ void ConnectionsDock::_signal_menu_about_to_popup() {
 		}
 	}
 
-	signal_menu->set_item_disabled(slot_menu->get_item_index(DISCONNECT_ALL), disable_disconnect_all);
+	signal_menu->set_item_disabled(signal_menu->get_item_index(DISCONNECT_ALL), disable_disconnect_all);
```
Comment (dalexeev) [inline editor/connections_dialog.cpp:1141]: I haven't tested, but this is obviously a bug.

**Review by Calinou — APPROVED** [review]
Tested locally, it works as expected.

File: editor/connections_dialog.cpp (hunk 1 above)
Comment (MewPurPur) [reply editor/connections_dialog.cpp:1147]: Might be the cause of "Disconnect" sometimes only working from the button on the bottom. Hooray!

**Review by MewPurPur — APPROVED** [review]

**Comment by MewPurPur** [comment]
(missed a review comment, I didn't test it but the code looks excellent to me)

**Pull request merged**